src/imageuploadcommand.js

```diff
@@ -28,6 +28,7 @@ export default class ImageUploadCommand extends Command {
 	 * @fires execute
 	 * @param {Object} options Options for executed command.
 	 * @param {File} options.file Image file to upload.
+	 * @param {module:engine/model/position~Position} [options.insertPosition] Position of the inserted image.
```

Comment:
Wouldn't `insertAt` be a better name? `insert<Subject>` means "insert the subject" for me.